### DISCUSSION TITLE
Username and password

### DISCUSSION THREAD

**Discussion by GeErghon** [jello1974/duosidaEV-home-assistant#7] [Q&A] [answered]
Hi, I just installed the component and it is asking for a username and password. I don't have it, I just have the network password. How can I get it. thanks

**Comment by jello1974** [comment]
First you have to install DS Charge app and pair with your device.
The user account and password you use with the app are the ones you use to controll it from HA.

**Reply by GeErghon** [reply]
I had already did that, but it does not ask for any username\password. My Duosida model is SES-32 and the app that I have is this
![Screenshot](https://github.com/jello1974/duosidaEV-home-assistant/assets/36460158/edbf950e-41a1-497e-a4f3-486f508dcf0d)

Thanks for your fast response 👍

**Comment by jello1974** [comment]
You took the wrong app. The right app is DS Charge with blue logo.

**Reply by GeErghon** [reply, marked as answer]
Worked like a charm. Nice work

**Reply by jello1974** [reply]
Happy to help
